kiwiapp-public/src/components/common/Container.tsx

```diff
@@ -0,0 +1,11 @@
+import { cn } from "@/utils/common.utils";
+import type { FC, ReactNode } from "react";
+
+interface ContainerProps {
+  children: ReactNode;
+  className?: string;
+}
+
+export const Container: FC<ContainerProps> = ({ children, className }) => (
+  <div className={cn("mx-auto max-w-[1300px]", className)}>{children}</div>
```

Comment:
1300px isn't standard. Use standard breakpoints (e.g. `max-w-screen-xl` is 1280px)
https://tailwindcss.com/docs/responsive-design#overview